src/main/resources/templates/gcc/activities-class-hourly-schedule.html

```diff
@@ -1,18 +1,51 @@
 <!DOCTYPE html>
 <html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
-<head th:replace="~{fragments/head :: head(title='TEMPLATE')}"></head>
+<head th:replace="~{fragments/head :: head(title=#{activities-class-hourly-schedule.title})}"></head>
+<script th:inline="javascript">
+  function changeVisibiliryActivitiesWeekly() {
+    let locale = $('#activitiesClassHoursSameEveryDay')[0].value
+    let locale = $('#activitiesWeekly')[0].value
+    if
+
+
+    let formAction = [[ ${formAction} ]] + "?lang=" + ("Spanish" === locale ? 'es' : 'en');
+    let form = $('form')[0]
+    form.setAttribute('action', formAction);
+  }
+</script>
```

Comment:
We can also just add a handler here without making FFL changes
```suggestion
<script th:inline="javascript">
    $(document).ready(function () {
        $('#activitiesClassHoursSameEveryDay-true').change(function () {
            $('#grouped-days').toggleClass('hidden');
            $('#individual-days').toggleClass('hidden');
        });
    });
</script>
```